allennlp/service/servable/models/simple_tagger.py

```diff
@@ -0,0 +1,61 @@
+import codecs
+import os
+from tempfile import gettempdir
+
+from allennlp.common import Params
+from allennlp.data.dataset_readers.sequence_tagging import SequenceTaggingDatasetReader
+from allennlp.data.fields import TextField
+from allennlp.data.token_indexers import SingleIdTokenIndexer
+from allennlp.data.tokenizers import WordTokenizer
+from allennlp.data.vocabulary import Vocabulary
+from allennlp.models.simple_tagger import SimpleTagger
+from allennlp.service.servable import Servable, JSONDict
+
+# Simple Tagger Model
+class SimpleTaggerServable(Servable):
```

Comment:
Yeah, that makes sense, though it's still a _class_ of `Models` that each of these can serve, not just a single model.  I think you'd still want to take `Params` as an argument to this, and maybe restrict the kinds of valid `Params` that are accepted.